plasmapy/particles/tests/test_ionization_collection.py

```diff
@@ -650,7 +650,7 @@ def test_getitem_two_indices(self, indices):
         integer_charge = indices[1]
```

Comment:
To be consistent, it would be a good idea to rename cases like this one to `charge_number`.

Reply:
Good catch!  I had thought I had found all of those...